internal/typename/types.go

```diff
@@ -0,0 +1,45 @@
+package typename
```

Comment:
I would make this an internal package of `typename` itself. Even though it's already internal, this stuff is only for the stuff in `typename`. 

That would mean this file would be called `internal/typename/internal/typenametest/types.go`.

This applies to any types used for testing and the `Declare()` method, etc.

Reply:
Should be fixed in https://github.com/dogmatiq/configkit/pull/115/commits/0bebaa4c6179a3db77be5a9e858cca5a519190f7